README.md

```diff
@@ -66,4 +66,17 @@ To launch a self contained integration test, run:
 
 ```
 make test-integration
-```
\ No newline at end of file
+```
+
+Adding new metrics to send via telemeter
+-----------
+
+Clone repository locally and in the root of the directory make the following changes:
```

Comment:
Not sure if we want to link the internal docs, or not?

Reply:
I'd say let's add a link here, these docs are gated anyways.